qualang_tools/multi_user/README.md

```diff
@@ -0,0 +1,30 @@
+# Multi-user tools
```

Comment:
Nice,
We also need to add it to the main readme file

Reply:
added